README.md

```diff
@@ -65,30 +105,19 @@ Follow the instructions below to use the Thali Cordova Plugin on Android and iOS
 #### Android
 
 1. Add the Android platform
-   * cd ThaliTest
-   * cordova platform add android
+   * `cd ThaliTest`
+   * `cordova platform add android`
 2. Fix manifest min-sdk issue
-   * go to ThaliTest\platforms\android and in AndroidManifest.xml change android:minSdkVersion="10" to
-    android:minSdkVersion="16"
+   * go to `ThaliTest\platforms\android` and in `AndroidManifest.xml` change `android:minSdkVersion="10"` to
+    `android:minSdkVersion="16"`
 3. Add the plugin
-   * cordova plugin add https://github.com/thaliproject/Thali_Cordovaplugin
+   * `cordova plugin add https://github.com/thaliproject/Thali_Codovaplugin`
 4. Fix issue on can not replace existing file
-   * from ThaliTest\plugins\org.thaliproject.p2p\src\android\java\io\jxcore\node copy 
-   the JXcoreExtension.java to ThaliTest\platforms\android\src\io\jxcore\node 
+   * from `ThaliTest\plugins\org.thaliproject.p2p\src\android\java\io\jxcore\node` copy
+   the `JXcoreExtension.java` to `ThaliTest\platforms\android\src\io\jxcore\node`
 (replace file, or copy the plug-in code and add it to existing file)
-5. Add example code into the app
```

Comment:
No, because there is nothing in the Thali Cordova Plugin itself that has any .gz files in it.  The postcard app does, but not the plugin